users_ldap_mail/users_ldap_model.py

```diff
@@ -19,32 +19,29 @@
 #
 ##############################################################################
```

Comment:
Please convert all headers to the new style - https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3